test/fixtures/served/amp-dynamic-css-classes.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async custom-element="amp-dynamic-css-classes" src="/base/dist/v0/amp-dynamic-css-classes-0.1.max.js"></script>
```

Comment:
So, this way the custom element itself is not involved, right? We need to review this since our plan was to eventually strip any custom-element script that has no actual elements in the document.